lib/blocklist.js

```diff
@@ -0,0 +1,48 @@
+/* Any copyright is dedicated to the Public Domain.
```

Comment:
From mozilla-central: the tests are public domain, other stuff is MPL.

I think Public domain for tests is because you can take the tests and run against your own implementation of something (maybe?)